implementations/rust/ockam/ockam_command/src/tcp/inlet/create.rs

```diff
@@ -105,10 +104,9 @@ async fn rpc(ctx: Context, (opts, mut cmd): (CommandGlobalOpts, CreateCommand))
     Ok(())
 }
 
-fn alias_parser(arg: &str) -> anyhow::Result<String> {
-    ensure! {
-        !arg.contains(':'),
-        "an inlet alias must not contain ':' characters"
+fn alias_parser(arg: &str) -> Result<String> {
+    if arg.contains(':') {
```

Comment:
Why `panic` here and not return an `Err`?

Reply:
I thought about that, but I didn't want to change the behaviour. It doesn't make much sense, so I'm going change it.